usefulUtilities/hiFiCalendar/entities/calendarEntities.json

```diff
@@ -48,54 +50,55 @@
     },
     "damping": 0,
     "angularDamping": 0,
-    "collisionless": true,
```

Comment:
why is this entity no longer collisionless?

Reply:
That is not my change. I pulled this from the www content but will add the collisionless property.